daprdocs/content/en/developing-applications/building-blocks/actors/actors-features-concepts.md

```diff
@@ -0,0 +1,93 @@
+---
+type: docs
+title: "Actors features and concepts"
```

Comment:
I am not sure how you split this section out. It seems like this is supposedly more detail on the features, but then there is also detail in the overview. What was this new document intending on covering?